js/viz/core/themes/material.js

```diff
@@ -61,7 +61,7 @@ registerTheme({
         paddingTopBottom: 6,
         arrowLength: 0,
         location: "edge",
-        color: "#616161",
+        color: "#fff",
```

Comment:
White text on the white background - it is the worst idea I have ever seen. Right color described in the documentation https://material.io/design/components/tooltips.html#spec.